bigquery/bqml/data_scientist_tutorial_test.py

```diff
@@ -16,29 +16,34 @@
 from google.cloud import bigquery
 # [END bqml_data_scientist_tutorial_import_and_client]
 import pytest
+import uuid
 
 # [START bqml_data_scientist_tutorial_import_and_client]
 client = bigquery.Client()
+# We use a unique dataset ID for this example to avoid collisions with
+# other invocations of this tutorial.  In practice, you could leverage
+# a persistent dataset and not create/destroy it with each invocation.
+dataset_id = "bqml_tutorial_{}".format(str(uuid.uuid4().hex))
```

Comment:
Would it be better to move this outside the region tag?

Reply:
It would be better, but much of the subsequent SQL requires rewriting to use the generated ID so I thought it aid understanding why all the SQL has python format strings.